app/questionnaire/questionnaire_schema.py

```diff
@@ -78,6 +78,9 @@ def get_blocks(self):
     def get_block(self, block_id):
         return self._blocks_by_id.get(block_id)
 
+    def is_block_valid(self, block_id):
+        return bool(self._blocks_by_id.get(block_id))
```

Comment:
Where possible try to use the class's public methods

Reply:
Done.